src/lib/appContainer.ts

```diff
@@ -94,19 +94,26 @@ export interface IAppContainer {
   govNotifyGateway: IGovNotifyGateway;
   accountHolderApiGateway: IAccountHolderApiGateway;
   userSessionGateway: IUserSessionGateway;
+  draftRegistrationGateway: DraftRegistrationGateway;
 }
 
 // "overrides" is spread over the default appContainer at the bottom of this method to enable injecting mocks et al.
 export const getAppContainer = (overrides?: IAppContainer): IAppContainer => {
   return {
     /* Simple use cases */
-    getCachedRegistration: getCachedRegistration,
-    saveCachedRegistration: saveCachedRegistration,
     clearCachedRegistration: clearCachedRegistration,
     deleteCachedUse: deleteCachedUse,
-    addNewUseToDraftRegistration: addNewUseToDraftRegistration,
```

Comment:
These use cases needed to have access to the container to get the gateway, so replaced with the `get` pattern below.